client/landing/stepper/declarative-flow/internals/hooks/use-launchpad-decider/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { updateLaunchpadSettings } from '@automattic/data-stores';
+import { useExperiment } from 'calypso/lib/explat';
+
+export const LAUNCHPAD_EXPERIMENT_NAME = 'calypso_onboarding_launchpad_removal_test_2024_08';
+
+interface Props {
+	exitFlow: ( path: string ) => void;
+	navigate: ( path: string ) => void;
+}
+
+interface PostFlowUrlProps {
+	flow: string;
+	siteId: string | number | null;
+	siteSlug: string;
+}
+
+interface SiteProps {
+	siteId: string | number | null;
+	siteSlug: string | null;
+}
+
+export const useLaunchpadDecider = ( { exitFlow, navigate }: Props ) => {
+	const [ isLoadingExperiment, experimentAssignment ] = useExperiment( LAUNCHPAD_EXPERIMENT_NAME );
+
+	const shouldShowCustomerHome =
+		! isLoadingExperiment &&
+		( 'treatment' === experimentAssignment?.variationName ||
+			// for testing/development purposes we can use sessionStorage to force the treatment
+			'treatment' === window.sessionStorage.getItem( 'launchpad_experiment_variation' ) );
```

Comment:
It may be worth wrapping this access in a function so we can make it a no-op and/or change the key name easily.

Reply:
Absolutely agree. Done!